orders.go

```diff
@@ -215,6 +220,36 @@ func (repository orderRepository) Create(order shopify.Order) (shopify.Order, er
 	return response.Order.ToShopify(), nil
 }
 
+func (repository orderRepository) CreateFulfilled(order shopify.Order) (shopify.Order, error) {
```

Comment:
How do we make sure that this function creates fulfilled orders? I would expect some hard coded values here that set the fulfilment status to `fulfilled`, regardless of what's passed down to this function 🙂

Reply:
Hi @ezelopes-mohc  - something like this?

bodyData.Order.FulfillmentStatus = "fulfilled"

Reply:
Yes, can you please write a comment explaining why? 🙂